inc/views/base/member/register.twig

```diff
@@ -0,0 +1,370 @@
+{% extends 'layouts/master.twig' %}
+
+{% block head %}
+    <title>{{ mybb.settings.bbname }} - {{ lang.registration }}</title>
+    <script type="text/javascript" src="{{ asset_url('jscripts/validate/jquery.validate.min.js') }}"></script>
+{% endblock head %}
+
+{% block body %}
+    <form action="member.php" method="post" id="registration_form">
+        <input type="text" style="visibility: hidden;" value="" name="regcheck1" />
```

Comment:
Shouldn't this just be `<input type="hidden">` or is there a particular reason for this approach?

Reply:
I believe that's a spam bot check.